development/php/Dockerfile

```
@@ -1,10 +1,13 @@
-FROM php:8.4-cli-bookworm
+FROM php:8.4-cli-bookworm AS base
 
 COPY --from=composer/composer:2.9-bin /composer /usr/bin/composer
 
 RUN apt-get update --quiet --assume-yes \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)